samples/bluetooth/peripheral_hr/CMakeLists.txt

```diff
@@ -1,4 +1,14 @@
 cmake_minimum_required(VERSION 3.13.1)
+
+# Some boards could run this sample using frdm_kw41z shield, enforce
+# -DSHIELD option in case they are used
+if(BOARD STREQUAL mimxrt1020_evk OR
+   BOARD STREQUAL mimxrt1050_evk OR
+   BOARD STREQUAL mimxrt1060_evk OR
+   BOARD STREQUAL frdm_k64f)
```

Comment:
Well that's unfortunate. :(

Reply:
Eventually we are going to need something more sophisticated than CMake variables to give input to DT.

Like a separate Kconfig root/tree for DT for instance.